third_party/terraform/utils/provider.go.erb

```diff
@@ -373,7 +373,6 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 	config.<%= product_name -%>BasePath = d.Get(<%= product_name -%>CustomEndpointEntryKey).(string)
 	<% end -%>
 
-	// Handwritten Products / Versioned / Atypical Entries
```

Comment:
Intentional?

Reply:
...I have no idea why this line is gone. let's bring it back!